spec/truffle.mspec

```diff
@@ -103,6 +103,7 @@ class MSpecScript
 
   set :ruby25, [
     "spec/ruby/core/kernel/yield_self_spec.rb",
+    "spec/ruby/core/enumerable/any_spec.rb",
```

Comment:
Could you add the specs of all/one/none here too?

Reply:
Oops, forgot those. Done.